.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!-- Write a description of what the PR solves and how -->
+

```

Comment:
Not something that should be changed, but I would add (as an example, still) one or two PR examples, so people can see some references on how to write their description. It may be helpful sometimes.

Reply:
Good point. I'll look through later and find something to link